soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -3717,12 +3716,11 @@ void DrawRandoEditor(bool& open) {
             return;
         }
 
-        bool disableEditingRandoSettings = CVar_GetS32("gRandoGenerating", 0) ||
-                                           CVar_GetS32("gOnFileSelectNameEntry", 0);
+        bool disableEditingRandoSettings = CVar_GetS32("gRandoGenerating", 0) || CVar_GetS32("gOnFileSelectNameEntry", 0);
+        bool disableZeldaRelatedOptions = CVar_GetS32("gRandomizeSkipChildZelda", 0) || disableEditingRandoSettings;
```

Comment:
So was the problem with not checking the generating flag that these would stay enabled during seed generation?

Reply:
Yes, because I'm setting the `ImGuiItemFlags_Disabled` further down in ImGui stuff. So if `disableZeldaRelatedOptions` was false, but `disableEditingRandoSettings` was true, that small part would still not be disabled.

That said, I just noticed when double checking that these options are now even more transparent than other options because the Alpha style is applied twice to those elements, so I guess I still have some fixing to do D:

Reply:
I changed it around now. It's now wrapped in if statements checking for if `disableEditingRandoSettings` is false to even get to the part to disable the checkboxes specifically. This also does fix the "double alpha" problem.